contracts/zero-ex/contracts/test/TestFundRecoveryFeature.sol

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.6;
+pragma experimental ABIEncoderV2;
+import "../src/features/FundRecoveryFeature.sol";
+
+contract TestFundRecoveryFeature is FundRecoveryFeature {
```

Comment:
it looks like you're not actually using this contract in the tests, so might as well delete